src/views/TimezoneModal.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Select time zone") }}</ion-title>
+    </ion-toolbar>
+    <ion-toolbar>
+      <ion-searchbar @ionFocus="selectSearchBarText($event)" :placeholder="$t('Search time zones')"  v-model="queryString" @ionInput="findTimeZone()"></ion-searchbar>
```

Comment:
Use self-closing tag

Reply:
Updated the code to use self closing tags.